packages/bpk-component-floating-notification/README.md

```diff
@@ -0,0 +1,28 @@
+# bpk-component-floating-notification
+
+> Backpack example component.
+
+## Installation
+
+```sh
+npm install bpk-component-floating-notification --save-dev
+```
+
+## Usage
+
+```js
+import React from 'react';
+import BpkFloatingNotification from 'bpk-component-code';
+
+export default () => <BpkFloatingNotification />;
+```
+
+## Props
+
+| Property  | PropType | Required | Default Value |
+| --------- | -------- | -------- | ------------- |
+| className | string   | false    | null          |
+| ctaText   | string   | false    | null          |
+| darkMode  | bool     | false    | false         |
+| icon      | bool     | false    | false         |
```

Comment:
For the icon we need to be able to say what icon is rendered. In the specs its a heart but in the search controls one its a notification icon

Reply:
This has been changed. The consumer can now pass in an icon in